committing.rst

```diff
@@ -226,30 +250,6 @@ a new paragraph(s) can be added to explain in proper depth what has happened
 understands the justification for the change).  Also, if a non-core developer
 contributed to the resolution, it is good practice to credit them.
 
-Hooks
-'''''''''''''''
-
-Special hooks have been added to the repository to enable notifying the `issue tracker`_
-of a commit related to an issue.
-
-A commit message can mention one or several issues in one of the following
```

Comment:
It would be nice to keep a tweaked version of this section. I used it as a reference for new core developers before.